CONTRIBUTION.MD

```diff
@@ -11,8 +11,18 @@ dzil authordeps --missing | sudo cpanm
 
 After changing some code run:
 
+
```

Comment:
These should all be handled by `dist test --author`. I have started working on these author tests. Please checkout the devel branch.

The tests are in awful shape and need work badly. We need to whip them into shape.